### PR TITLE
Minor cosmetic changes to header and footer

### PR DESCRIPTION
**Pull request by niden**

### CODE REVIEW

File: es/index.rst [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-===========
+Bienvenido
+==========
 
 Phalcon es una nueva alternativa en frameworks para PHP. Nuestra misión es proporcionarte una herramienta avanzada para desarrollar
 sitios y aplicaciones web sin preocuparte por el rendimiento del framework.
 
-¿Qué es Phalcon?
-----------------
+¿Qué es Phalcon
```
Comment (sergeyklay) [inline es/index.rst]: http://studyspanish.com/grammar/lessons/quest

Comment (andresgutierrez) [inline es/index.rst]: hahaha

Comment (sergeyklay) [inline es/index.rst]: :)

Comment (sergeyklay) [inline es/index.rst]: recuerdo sus clases

**Pull request merged**